gc/base/MemoryPoolAddressOrderedListBase.hpp

```diff
@@ -48,6 +48,7 @@ class MM_MemoryPoolAddressOrderedListBase : public MM_MemoryPool
 	MM_HeapLinkedFreeHeader ** _referenceHeapFreeList;
 	MM_SweepPoolState* _sweepPoolState;	/**< GC Sweep Pool State */
 	MM_SweepPoolManagerAddressOrderedListBase* _sweepPoolManager;		/**< pointer to SweepPoolManager class */
+	MM_HeapLinkedFreeHeader *_lastFreeEntry;							/**< address of the last free entry in the pool; valid after compact; NOT maintained during allocation */ 
```

Comment:
Does this make sense to move up to the super class instead of having the default case of NULL?